hack/lib/test.sh

```diff
@@ -34,7 +34,7 @@ kube::test::get_object_assert() {
 
   res=$(eval kubectl ${args} get "${kube_flags[@]}" $object -o go-template=\"$request\")
 
-  if [[ "$res" =~ ^$expected$ ]]; then
+  if [[ "$res" =~ ^$expected ]]; then
```

Comment:
thats  a really good example.... i guess the semantics of this function needs to be EXACT MATCHing.